docs/configuration.md

```diff
@@ -149,39 +202,43 @@ ga_tracking_anonymize_ip: true # Use GDPR compliant Google Analytics settings (t
 ## Document collections
 
 By default, the navigation and search include normal [pages](https://jekyllrb.com/docs/pages/).
-Instead, you can also use [Jekyll collections](https://jekyllrb.com/docs/collections/) which group documents semantically together.
+You can also use [Jekyll collections](https://jekyllrb.com/docs/collections/) which group documents semantically together.
 
-For example, put all your documentation files in the `_docs` folder and create the `docs` collection:
+For example, put all your test files in the `_tests` folder and create the `tests` collection:
```

Comment:
Just curious - is there a reason that the default is changed to `tests` instead of `docs` (and some of the default options are different)?

Reply:
Yes, I think so: having a folder named `_docs` for creating a collection in the `docs` folder seems potentially confusing and risky when one might already have *manually* created files in `docs`! What happens if *both* folders contain an (html) file with the same name? I'm not sure the Jekyll developers envisaged that use case, and it's unclear (at least to me) which file would 'win' in the resulting website. Even if it's obvious to Jekyll experts, or stated somewhere in the Jekyll docs, I can see no motivation for *encouraging* our users to name a collection folder `_docs` (especially when we also encourage them to put their source files in `docs`!).

When collections were first introduced in Just the Docs, users who wanted to use a collection needed to move all their source files for ordinary pages into collection folders. Then there were no longer any source files in `docs`, so no confusion could arise. One of the PRs incorporated in the present `combination` is #448, allowing ordinary pages and collections to coexist. I personally find that feature a useful enhancement of our support for collections, and I don't recall any objections being raised to it.

A less significant motivation was in connection with adding tests for intra-collection navigation in the current repo.

Reply:
I see, thanks for the context - much appreciated!